### PR TITLE
addressing qrda issues and adding tests

### PR DESCRIPTION
**Pull request by edeyoung**
Added a significant amount of tests and addressed qrda r5 issues or added attributes onto qrda r5 templates that weren't there previously.

Pull requests into Health Data Standards require the following. Submitter and reviewer should :white_check_mark: when done. For items that are not-applicable, note it's not-applicable ("N/A") and :white_check_mark:.
 
**Submitter:**
- [x] This pull request describes why these changes were made.
- [x] This PR is into the correct branch.
- [x] Internal ticket for this PR: https://jira.mitre.org/browse/BONNIE-1608
- [x] Internal ticket links back to this PR
- [x] Code diff has been done and been reviewed (it **does not** contain: additional white space, not applicable code changes, debug statements, etc.)
- [x] Tests are included and test edge cases
- [x] Tests have been run locally and pass
- [x] Code coverage has not gone down and all code touched or added is covered.
  - master_bonnie: 90.49
  - bonnie-1608_qrda_fixes: 90.59
 
**Cypress Reviewer:**
 
Name: Dave Czulada
- [x] Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
- [x] The tests appropriately test the new code, including edge cases
- [x] You have tried to break the code
 
**Bonnie Reviewer (really Cypress Reviewer 2):**
 
Name: Michael O'Keefe
- [x] Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
- [x] The tests appropriately test the new code, including edge cases
- [x] You have tried to break the code

### CODE REVIEW

File: templates/cat1/r5/_2.16.840.1.113883.10.20.24.3.105.cat1.erb [hunk 1]
```diff
@@ -25,10 +25,9 @@
...
         </effectiveTime>
-        <% if entry.administrationTiming.present?
-          period = entry.administrationTiming['period'] %>
+        <% if entry.administrationTiming.present? %>
           <effectiveTime xsi:type="PIVL_TS" institutionSpecified="true" operator="A">
-            <period value="<%= period['value']%>" unit="<%= period['unit']%>"/>
+            <period value="<%= entry.administrationTiming['scalar']%>" unit="<%= entry.administrationTiming['units']%>"/>
           </effectiveTime>
         <% end %>
         <% if entry.route.present? -%>
```
Comment (dczulada) [inline templates/cat1/r5/_2.16.840.1.113883.10.20.24.3.105.cat1.erb]: in the code entry.dose[:value].  the dose appears to now be saved as entry.dose[:scalar]

```
<% if entry.dose.present? -%>
   <doseQuantity value="<%= entry.dose[:value]%>"/>
<% end -%>
```

File: test/unit/export/cat_1_qrda_r5_test.rb [hunk 2]
```diff
@@ -0,0 +1,927 @@
...
+      component_node2 = component_nodes.xpath("./xmlns:observation/xmlns:code[@code='11218-5']/../..")
+      assert_equal 1, component_node1.count
+      assert_equal 1, component_node2.count
+
+      # component 1
+      # Component: Diabetic Nephropathy, 56
+      component_result_node = component_node1.xpath("./xmlns:observation/xmlns:value")
+      assert_equal 1, component_result_node.count
+      assert_equal "56", component_result_node.xpath("./@value").inner_text
+      assert_equal "", component_result_node.xpath("./@unit").inner_text
```
Comment (dczulada) [inline test/unit/export/cat_1_qrda_r5_test.rb]: Does this want to be "1"?

I believe this is related to the change made in view_helper

**Review by dczulada — DISMISSED** [review]

**Review by okeefm — APPROVED** [review]
Looks good. Verified that the templates match the QRDA Cat I R5 IG.
